components/EmotionCardContent.js

```diff
@@ -1,25 +1,73 @@
 import styled from "styled-components";
 import { emotionsIcons } from "./EmotionIcons";
-import formatDate from "./TransformDateTime";
 
 export default function EmotionCardContent({ emotion }) {
   return (
-    <>
-      <h2>{emotion.emotionType}</h2>
+    <StyledEmotionCardContent>
       <StyledEmojiIcon>{emotionsIcons[emotion.emotionType]}</StyledEmojiIcon>
-      <p>Intensity: {emotion.intensity}</p>
-      <p>Notes: {emotion.notes}</p>
-      <StyledDateTime>{formatDate(emotion.dateTime)}</StyledDateTime>
-    </>
+      <StyledEmotionType>{emotion.emotionType}</StyledEmotionType>
+      <StyledIntensity>{emotion.intensity}</StyledIntensity>
+      <StyledNotes>{emotion.notes}</StyledNotes>
+    </StyledEmotionCardContent>
   );
 }
 
+const StyledEmotionCardContent = styled.div`
+  display: grid;
+  grid-template-columns: 60px 1fr auto;
+  grid-template-rows: auto auto;
+  grid-template-areas:
+    "emoji emotionType intensity"
+    "emoji notes intensity";
+  align-items: center;
+  padding: 5px;
+
+  @media (max-width: 374px) {
+    grid-template-columns: 1fr;
+    grid-template-rows: auto auto auto auto;
+    grid-template-areas:
+      "emoji"
+      "emotionType"
+      "notes"
+      "intensity";
+  }
+`;
+
 const StyledEmojiIcon = styled.span`
-  align-self: flex-end;
-  width: 40px;
-  height: 40px;
+  grid-area: emoji;
+  align-self: self-start;
+  width: 50px;
+  height: 50px;
+  color: #313366;
+
+  svg {
+    stroke-width: 0.8px;
+  }
+`;
+
+const StyledEmotionType = styled.p`
+  grid-area: emotionType;
+  margin: 0;
+  padding-left: 28px;
+  font-weight: bold 600;
```

Comment:
Fix Typo: Separate `bold` from the numeric font weight
```suggestion
  font-weight: 600;
```